terraform/datadog/datadog.tf

```diff
@@ -0,0 +1,29 @@
+variable "datadog_api_key" {}
+variable "datadog_app_key" {}
+variable "env" {}
+
+provider "datadog" {
+    api_key = "${var.datadog_api_key}"
+    app_key = "${var.datadog_app_key}"
+}
+
+resource "datadog_monitor" "router" {
+  name = "${format("%s router hosts", var.env)}"
+  type = "service check"
+  message = "Missing router hosts in environment {{host.environment}}. Notify: @the-multi-cloud-paas-team@digital.cabinet-office.gov.uk"
```

Comment:
After deploying, this check (along with an ntp sync check) is present in the datadog webui, but it looks a bit strange.
Can you explain when {{host.environment}} is supposed to be replaced?
And what the `Notify: @the-multi-cloud-paas-team@digital.cabinet-office.gov.uk` achieves?